spec/fabricators/workshop_fabricator.rb

```diff
@@ -43,7 +43,7 @@
   end
 end
 
-Fabricator(:workshop_no_spots, class_name: :workshop) do
+Fabricator(:workshop_barring_spots, class_name: :workshop) do
```

Comment:
@notapatch I actually prefer `workshop_no_spots` name here, barring might be hard to understand for some contributors (I myself am an English as second language speaker).

Reply:
RE: second language: I did not know that.
I'll aim to keep the language straight forward - I didn't think about that despite talking to two Fins this week.

Reply:
I've pressed "resolve conversation" as you probably know by now I have no idea what all these buttons do.